pdf-generator/src/lib/validation.ts

```
@@ -0,0 +1,342 @@
+  }
+
+  return (
+    text
+      // Remove HTML tags
+      .replace(/<[^>]*>/g, '')
+      // Remove potential script content
+      .replace(/javascript:/gi, '')
+      // Remove excessive whitespace
+      .replace(/\s+/g, ' ')
+      // Remove control characters
+      .replace(/[\x00-\x1F\x7F]/g, '')
```

Biome (2.1.2)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)